modules/index.js

```diff
@@ -0,0 +1,59 @@
+import {
+  BookFun,
+  form,
+  titleInput,
+  authorInput,
+  addBtn,
+  bookList,
+} from './book.js';
+
+import { DateTime } from './luxon/src/luxon.js';
+
+const clockElement = document.getElementById('clock');
+const clock = () => {
+  clockElement.textContent = DateTime.now().toISO();
+};
+setInterval(clock, 1000);
+
+const books = new BookFun();
+
+window.addEventListener('load', () => {
+  books.showBook();
+});
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const book = { title: titleInput.value, author: authorInput.value };
+
+  if (titleInput.value !== '' && authorInput.value !== '') {
+    addBtn.disabled = false;
+    books.addBook(book);
+    titleInput.value = '';
+    authorInput.value = '';
+    books.showBook(book);
+  } else {
+    addBtn.disabled = true;
+  }
+});
+
+bookList.addEventListener('click', (e) => {
+  if (e.target.className.includes('remove-btn')) {
+    const targetEl = e.target;
+    books.removeBook(targetEl);
+  }
+});
+
+const navMenus = document.querySelectorAll('nav a');
+const contentEls = document.querySelectorAll('.contents');
+
+for (let i = 0; i < navMenus.length; i += 1) {
+  navMenus[i].addEventListener('click', () => {
```

Comment:
- [ ] Kindly use forEach instead of traditional for loop because you are making your project using ES6 and forEach is introduced in ES6 for iteration purpose :blush: